### PR TITLE
Rename getting-started.md to README.md.

### PR DESCRIPTION
**Pull request by Random-Liu**
Rename getting-started.md to README.md, so that it will be shown by github directly.
/cc @abhi 

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ sudo cri-containerd -v 2 --alsologtostderr
 ```shell
 CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/cri-containerd.sock' ./hack/local-up-cluster.sh
 ```
-For a multi node cluster installer and bring up steps using ansible and kubeadm refer to [Getting started](contrib/ansible/getting-started.md) guide.
+For a multi node cluster installer and bring up steps using ansible and kubeadm refer to [Getting started](contrib/ansible/README.md) guide.
```
Comment (mikebrow) [inline README.md]: also the link text .. did you want to change that to readme?

Comment (mikebrow) [reply README.md]: maybe:For a multi node cluster installer and bring up steps using ansible and kubeadm refer [here](contrib/ansible/README.md).

Comment (Random-Liu) [reply README.md]: Will do.

Comment (Random-Liu) [reply README.md]: Will do.

**Comment by Random-Liu** [comment]
@mikebrow Done.

**Comment by Random-Liu** [comment]
Document only change. Merged.

**Comment by abhi** [comment]
LGTM

**Pull request merged**